package.json

```diff
@@ -90,7 +90,6 @@
     "debug": "2.2.0",
     "minimist": "1.2.0",
     "ramda": "0.21.0",
-    "rollup": "0.37.0",
```

Comment:
After thinking about it i concur. However this may be a breaking change for those not in the know. Any suggestions for those that are sans `rollup` in their package.json @YuryStrozhevsky?